lxd/instance/drivers/driver_lxc.go

```diff
@@ -5273,9 +5273,18 @@ fi
 }
 
 // MigrateSend controls the sending side of a migration.
-func (d *lxc) MigrateSend(args instance.MigrateSendArgs) error {
+func (d *lxc) MigrateSend(args instance.MigrateSendArgs) (err error) {
 	d.logger.Info("Migration send starting")
-	defer d.logger.Info("Migration send stopped")
+	defer func() {
+		d.logger.Info("Migration send stopped")
```

Comment:
hrm, shouldnt we only be calling d.postMigrateSendCommon if the migration send completed successfully, can this be called at the bottom of the function before we return err?